core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/cluster/Cluster.java

```diff
@@ -241,6 +241,9 @@ public class Cluster implements ProvisionEntity, WorkspaceAwareResource, Cluster
     @Column(nullable = false)
     private Boolean embeddedDatabaseOnAttachedDisk = Boolean.FALSE;
 
+    @Column(columnDefinition = "TEXT")
+    private String dbSslCert;
```

Comment:
As discussed privately, we need one more field now, `boolean `dbSslEnabledForStack` (or simply `dbSslEnabled`, likely the "ForStack" suffix is redundant).